pjsip/include/pjsua2/call.hpp

```diff
@@ -2014,27 +2014,25 @@ class Call
      *
      * @param prm       Callback parameter.
      */
-    virtual void onCallReplaced(OnCallReplacedParam &prm)
-    { PJ_UNUSED_ARG(prm); }
+  virtual void onCallReplaced(OnCallReplacedParam &prm) { PJ_UNUSED_ARG(prm); }
```

Comment:
why does this have to be squeezed into one line?

Reply:
This was done by accident.